docs/rules/no-multiple-assertions-wait-for.md

```diff
@@ -0,0 +1,54 @@
+# Multiple assertions inside `waitFor` are not preferred (no-multiple-assertions-wait-for)
+
+## Rule Details
+
+This rule aims to ensure the correct usage of `expect` inside `waitFor`, in the way that they're intended to be used.
+When using multiples assertions inside `waitFor`, if one fails, you have to wait for a timeout before seeing it failing.
+Putting one assertion, you can both wait for the UI to settle to the state you want to assert on,
+and also fail faster if one of the assertions do end up failing
+
+Example of **incorrect** code for this rule:
+
+```js
+const foo = async () => {
+  await waitFor(() => {
+    expect(a).toEqual('a');
+    expect(b).toEqual('b');
+  });
+
+  // or
+  await waitFor(function() {
+    expect(a).toEqual('a')
+    expect(b).toEqual('b');
+  })
+};
+```
+
+Examples of **correct** code for this rule:
+
+```js
+const foo = async () => {
```

Comment:
Could you add same examples from incorrect block but corrected please? You are just using correct examples with a single expect, so it would be nice if incorrect examples can be added here showing how it should be fixed.